modules/aws/cnid-vpc/variables.tf

```diff
@@ -82,3 +82,15 @@ variable "master_sg_ingress_cidr" {
   default     = []
   type        = "list"
 }
+
+variable "master_sg_ssh_ingress_cidr" {
+  description = "A list of whitelisted ingress CIDRs to masters"
```

Comment:
Might make it better to have a more consise description for "ssh CIDRs" and what that means...